cves/CVE-2019-14232.yml

```diff
@@ -105,10 +113,14 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
+  code: true
   code_answer:
-  fix:
+    There were already tests for this part of the system, but nothing testing this
+    exact case.
+  fix: true
   fix_answer:
+    After applying the fix 2 tests were updated for new expected output and 2
+    new tests were added to check this specific case.
```

Comment:
Elaborating on how the tests now cover the vulnerability could help others better understand the vulnerability